src/main/java/org/candlepin/subscriptions/tally/billing/BillableUsageController.java

```diff
@@ -137,7 +147,7 @@ private BillableUsageCalculation calculateBillableUsage(
 
     return BillableUsageCalculation.builder()
         .billableValue(billableValue.getValue())
-        .remittedValue(totalRemittance.getValue())
+        .remittedValue(billableValue.to(new MetricUnit()).getValue())
         .remittanceDate(clock.now())
```

Comment:
This should be set to usage.getSnapshotDate() so that the remittance has the snapshot date as the remittancePendingDate.